### PR TITLE
PSR-767 bonds list page

### PR DESCRIPTION
**Pull request by narendrapaduchuri**

### CODE REVIEW

**Review by kberec — DISMISSED** [review]

File: app/controllers/testonly/BondsMongoController.scala
Comment (kberec) [inline app/controllers/testonly/BondsMongoController.scala:33]: I thought this is in @westwater 's  branch @narendrapaduchuri ?

File: .g8/textInputPage/app/pages/$className$Page.scala
Comment (kberec) [inline .g8/textInputPage/app/pages/$className$Page.scala:13]: Are these changes intentional @narendrapaduchuri ?

File: app/controllers/testonly/BondsMongoController.scala
Comment (narendrapaduchuri) [reply app/controllers/testonly/BondsMongoController.scala:33]: Yes @kberec , George also added this end point.. I will wait until George ticket merge to main

File: .g8/textInputPage/app/pages/$className$Page.scala
Comment (narendrapaduchuri) [reply .g8/textInputPage/app/pages/$className$Page.scala:13]: I think this is system generated ones, I haven't added that

**Review by kberec — APPROVED** [review]

**Review by kberec — APPROVED** [review]

**Pull request merged**
